server/libs/database/src/entities/exif.entity.ts

```diff
@@ -50,6 +50,9 @@ export class ExifEntity {
   @Column({ type: 'varchar', nullable: true })
   country!: string | null;
 
+  @Column({ type: 'char', length: 40, nullable: true })
+  pixelChecksum?: string | null; // sha1 checksum
```

Comment:
do we need to store both - a file/byte checksum (sha1) and an equivalent sha1 checksum of the decoded bytes? Seems duplicated :wink: 

If we really need it: let's use bytes instead of hex and `Uint8Array` instead of string

Reply:
`checksum` in asset is use for byte level and `pixelChecksum` for pixel level. As you can see, I file with the same content (in RGB buffer) but different in exif info, which can cause change of `checksum`. I think someone will need it, just reserve for future.

Reply:
I think if we need it for the future we can build it in later, there is no need for future guarding :)